### DISCUSSION TITLE
Could beanie support APM?

### DISCUSSION THREAD

**Discussion by jiewmeng** [BeanieODM/beanie#330] [Question]
I realised beanie is not supported by ElasticAPM (only pymongo is supported), was wondering if I were to want to instrument beanie with APM, how best can I approach this? Are there hooks I can use?

**Comment by roman-right** [comment]
I'm not sure if I got the question correctly. But for elastic there is a nice pydantic-based lib: https://github.com/RamiAwar/pydastic
Or do you need some log tracers from inside the beanie lib to elstic APM?
